### ISSUE TITLE
Couple of problems with rendered single-clip fades

### ISSUE THREAD

**Issue by thennicke** [jliljebl/flowblade#837]
Flowblade 2.4.0 Flatpak on Fedora 31

- Off by one error when deleting rendered fades (an extra frame is thrown in for some reason, shunting following clips to the right).
- Creating a rendered fade causes weird audio problems for all codecs except FFv1

**Comment by jliljebl** [comment]
Ok,will look.

**Comment by thennicke** [comment]
No problem. Thank you for all your hard work.
- Just a quick correction: It appears that it's an off-by-one error when deleting fade outs, and an off-by-two error for fade ins.

**Comment by thennicke** [comment]
I've done some more testing here.
Just as for the transitions bug, this bug is different depending whether an odd-sized or even-sized fade is created (I tested for both 5 and 6 frames in length). By the way, I found a very strange bug here: if we select the slip tool (7) and then move a newly-created fade around, we can move the fade past the edge of its clip. This happens for both odd and even sized fade ins and fade outs. For even sized fades it can go three frames past the edge of the clip, and for odd it can go two.
So there's something funny going on with the maths.
